tests/fixtures/StyledComponent.js

```diff
@@ -0,0 +1,22 @@
+export default {
+  $$typeof: Symbol('react.forward_ref'),
+  attrs: [],
+  componentStyle: {
+    rules: [
+      '\n  font-size: 20px;\n  line-height: 32px;\n  letter-spacing: .3;\n  margin: ',
```

Comment:
Nitpicking here, but the `rules` section could be simplified (exact styles are not relevant to the test).

Reply:
Sure, I just copied and pasted a real-life example of what a styled-component object looks like. I'll simplify it.